app/lib/server/methods/getMessages.js

```diff
@@ -7,10 +7,6 @@ Meteor.methods({
 	getMessages(messages) {
 		check(messages, [String]);
 
-		if (!Meteor.userId()) {
```

Comment:
Shouldn't this check be performed only when anonymous read is disabled? 🤔

Reply:
right, I was looking into this.. I **think** this is validated by the `canAccessRoom` method later on.